CHANGES.md

```diff
@@ -11,6 +11,7 @@
 - Fixing column width on Job and Logs Table.
 - Add specific tab query param key for GC tabs.
 - UI Improvements on SQL Editor.
+- Misc tiny tidy-ups.
```

Comment:
Probably after re-basing this has placed in the wrong section, it should be under "unreleased" 😄